liquibase-couchbase/src/test/java/liquibase/ext/couchbase/statement/RemoveDocumentsSqlPlusPlusQueryStatementTest.java

```diff
@@ -0,0 +1,78 @@
+package liquibase.ext.couchbase.statement;
+
+import com.couchbase.client.java.transactions.ReactiveTransactionAttemptContext;
+import com.couchbase.client.java.transactions.TransactionAttemptContext;
+import com.couchbase.client.java.transactions.TransactionGetResult;
+import liquibase.ext.couchbase.operator.BucketOperator;
+import liquibase.ext.couchbase.operator.ClusterOperator;
+import liquibase.ext.couchbase.operator.CollectionOperator;
+import liquibase.ext.couchbase.types.Id;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Flux;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import static common.constants.TestConstants.TEST_KEYSPACE;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+class RemoveDocumentsSqlPlusPlusQueryStatementTest {
+
+    private final TransactionAttemptContext transaction = mock(TransactionAttemptContext.class);
+    private final ReactiveTransactionAttemptContext reactiveTransactionAttemptContext =
+            mock(ReactiveTransactionAttemptContext.class);
+    private final ClusterOperator clusterOperator = mock(ClusterOperator.class);
+    private final BucketOperator bucketOperator = mock(BucketOperator.class);
+    private final CollectionOperator collectionOperator = mock(CollectionOperator.class);
+    private final List<String> docIds = new ArrayList<>();
+
+    @BeforeEach
+    public void configure() {
+        docIds.add("doc1");
+        docIds.add("doc2");
+        when(clusterOperator.getBucketOperator(TEST_KEYSPACE.getBucket())).thenReturn(bucketOperator);
+        when(bucketOperator.getCollectionOperator(TEST_KEYSPACE.getCollection(), TEST_KEYSPACE.getScope())).thenReturn(
+                collectionOperator);
+        when(clusterOperator.retrieveDocumentIdsByWhereClause(eq(TEST_KEYSPACE), any())).thenReturn(docIds);
```

Comment:
Pls check, I think here should be `retrieveDocumentIdsBySqlPlusPlusQuery` mock instead of `retrieveDocumentIdsByWhereClause`. And pls check docIds (it looks like no ID matches the search/delete condition now)